src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -129,6 +135,8 @@ public Exporter(final Config config, final FcrepoClient.FcrepoClientBuilder clie
                 bagdir.mkdirs();
                 this.bag = new Bag(new Version(0, 97));
                 this.bag.setRootDir(bagdir);
+                this.bag.getMetadata().putAll(bagConfig.getBagInfo());
+                //TODO where to add aptrust bag info from the bagConfig?
```

Comment:
I don't know if there is support for custom tag files in bagit-java — I looked at the API docs and don't see it anywhere.  So we may have to write our own.  Luckily, the format is just

```
key: value
```

So it shouldn't be hard to just iterate over the APTrust values and output them.